### PR TITLE
Disabled behavior added

### PR DESCRIPTION
**Pull request by fungio**

### CODE REVIEW

**Comment by arthurgouveia** [comment]
Hey, man! Thanks for that. Sorry for the long time to reply.

I'll be checking the pull request and merging stuff next week, ok?

**Comment by fungio** [comment]
No problem.

File: prettyCheckable.js [hunk 1]
```diff
@@ -36,24 +36,24 @@
...
+		if (input.attr('disabled') === undefined) {
+			if (input.attr('checked') !== undefined) {
+				input.removeAttr('checked').change();
+			} else {
+				input.attr('checked', 'checked').change();
+			}
+	
+			fakeCheckable.toggleClass('checked');
+		} else {
+			console.log('disabled');
```
Comment (arthurgouveia) [inline prettyCheckable.js:55]: Oops! :)

**Comment by arthurgouveia** [comment]
Hey, man! Sorry again for taking such a long time to do this.

I reversed the logic you added on the code and also added the possibility to enable/disabled checkables using methods. Because of the amount of code, I decided to just push it all instead of pulling your code and changing it.

Thanks for that, anyway. People have been asking me for this for quite some time. I hope it works fine :)

**Pull request closed without merging**